controllers/users.js

```diff
@@ -80,8 +81,13 @@ const login = async (req, res) => {
           data: access_token,
         });
       }
+    } else {
+      res.send({
+        error: 1,
+        message: "password don't match",
```

Comment:
This **else** part should not come here, 
**or**
you should change the message